src/categorical_algebra/FinCats.jl

```diff
@@ -124,6 +124,14 @@ abstract type FinCatGraph{G,Ob,Hom} <: FinCat{Ob,Hom} end
 """
 graph(C::FinCatGraph) = C.graph
 
+graph(C::OppositeCat) = op(graph(op(C)))
+function op(g::G) where G<:HasGraph 
```

Comment:
We should implement `Base.reverse` and `Base.reverse!` in our graphs module, following [Graphs.jl](https://juliagraphs.org/Graphs.jl/dev/core_functions/operators/#Base.reverse), and then have

```julia
graph(C::OppositeCat) = reverse(graph(op(C)))
```

This could be deferred until later, but if so, let's make an issue to track it.